### PR TITLE
Feature/fga/fix voip issues

### PR DESCRIPTION
**Pull request by ganfra**
This PR fixes multiple issues on Voip : 
#4028 
#4026 
#4019

### CODE REVIEW

File: vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt [hunk 1]
```diff
@@ -468,7 +468,6 @@ class NotificationUtils @Inject constructor(private val context: Context,
                         setSmallIcon(R.drawable.ic_call_answer)
                     }
                 }
-                // This is a trick to make the previous notification with same id disappear as cancel notification is not working with Foreground Service.
```
Comment (bmarty) [inline vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt:471]: So also remove the trick itself?

**Review by bmarty — COMMENTED** [review]
LGTM, just one question

File: vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt (hunk 1 above)
Comment (ganfra) [reply vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt:471]: Nope, it's not really a trick anymore, cancelNotification work, but only when stopForeground is called.

**Comment by DoM1niC** [comment]
I found a strange issue, I use Element develop on Android / Web and use this PR but I dunno where the issue is but If I dismiss a Call so my Element Client still ringing the Caller still rejected but not the React Client (Element Web)

File: vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt (hunk 1 above)
Comment (bmarty) [reply vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt:471]: OK, thanks

**Review by bmarty — APPROVED** [review]

**Comment by ganfra** [comment]
Thanks for commenting, I can't reproduce this behavior, not sure where it comes from, but shouldn't be from this PR.

**Pull request merged**